projects/addon-commerce/components/input-expire/test/input-expire.component.spec.ts

```diff
@@ -47,30 +47,4 @@ describe(`InputExpire`, () => {
         expect(testComponent.value).toBe(`12/12`);
         expect(input.value).toBe(`12/12`);
     });
-
-    describe(`Input correction`, () => {
-        it(`replaces 50/08 with 12/08`, () => {
```

Comment:
New mask connot be properly tested via units.
There were replaced by Cypress ones.